Sources/LiveKit/Core/Engine.swift

```diff
@@ -136,8 +140,6 @@ internal class Engine: MulticastDelegate<EngineDelegate> {
             completer.reset()
         }
 
-        connectStopwatch.clear()
```

Comment:
No need to clear this in the new `state` struct?

Reply:
yes should be ok 🙂